include/swift/AST/Types.h

```diff
@@ -1352,7 +1352,7 @@ class ParameterTypeFlags {
     NumBits = 3
   };
   OptionSet<ParameterFlags> value;
-  static_assert(NumBits < 8*sizeof(value), "overflowed");
+  static_assert(NumBits < 8*sizeof(OptionSet<ParameterFlags>), "overflowed");
```

Comment:
Dupe of https://connect.microsoft.com/VisualStudio/feedback/details/3116640/msvc-doesnt-support-sizeof-value-in-a-static-assert-in-a-class